test/models/comment_test.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class CommentTest < ActiveSupport::TestCase
+  test "sort comments ascendingly by created_at" do
```

Comment:
```suggestion
  test ". sorted" do
```

modelのテストはとかで良いかもです〜

Reply:
なるほど…！モデルテストでは、どのメソッドを呼んでいるのかが分かれば良く、テスト名は短く簡潔な方がわかりやすいですね。✨
Commit suggestionボタンから修正してみました、確認をお願いします！ 🙇‍♂️
eb19931